app/jobs/api_key_inactive_deletion_job.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'google_analytics_queries'
+
+class ApiKeyInactiveDeletionJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    active_api_keys = GoogleAnalyticsQueries.get_active_api_keys_from_ga(365)
```

Comment:
`365` is the number of days of some years, would it be better to fix the retention period to 365 days instead of 1 year? It would help making the query `1.year.ago` something like `365.days.ago` avoiding a potential off by 1.